config/settings.yml

```diff
@@ -1254,6 +1254,7 @@
       :ems_metrics_collector_worker:
         :defaults:
           :count: 2
+          :memory_threshold: 400.megabytes
```

Comment:
Sorry, in all of those words, I didn't say this default value wasn't missing... it's purposely being inherited from a parent class.